:arrow_up: (avrlibc): Upgrade to v2.1.0

ladislas opened the pull request.

ladislas committed:
:arrow_up: (avrlibc): Upgrade to v2.1.0

lukeadair commented:
Nooooooo! Any idea what's broken?

fauxpark commented on Formula/avr-gcc@9.rb:
```diff
@@ -40,9 +40,8 @@ class AvrGccAT9 < Formula
-    mirror "https://download-mirror.savannah.gnu.org/releases/avr-libc/avr-libc-2.0.0.tar.bz2"
-    sha256 "b2dd7fd2eefd8d8646ef6a325f6f0665537e2f604ed02828ced748d49dc85b97"
+    url "https://github.com/avrdudes/avr-libc/archive/refs/tags/avr-libc-2_1_0-release.tar.gz"
+    sha256 "97a5ffb25ad7ca0a241e8a73167fed8cbef6a2472822ccb697cb0df70f51e836"
```
```suggestion
    url "https://download.savannah.gnu.org/releases/avr-libc/avr-libc-2.1.0.tar.bz2"
    mirror "https://download-mirror.savannah.gnu.org/releases/avr-libc/avr-libc-2.1.0.tar.bz2"
    sha256 "0b84cee5c08b5d5cba67c36125e5aaa85251bc9accfba5773bfa87bc34b654e8"
```
https://github.com/osx-cross/homebrew-avr/pull/274#discussion_r894938601

ladislas committed:
Use avr-libc from savannah

the archives from GitHub are just a straight rip of the repo - ./bootstrap hasn't been run to generate the configure script etc.

The releases from Savannah appear to have this step already done.

Co-authored-by: Ryan <fauxpark@gmail.com>
